models/core.mdx

```diff
@@ -7,7 +7,7 @@ weight: 6
 no_list: true
 ---
 
-W&B Core is the foundational framework supporting [W&B Models](/models/) and [W&B Weave](/guides/weave/), and is itself supported by the [W&B Platform](/platform/hosting/). 
+W&B Core is the foundational framework supporting [W&B Models](/models/) and [W&B Weave](/weave), and is itself supported by the [W&B Platform](/platform/hosting/). 
```

Comment:
We should delete this file and redirect to /models/
All of the `core` subdirectories have been disbursed and will get their own redirects.